test/jellyfish_web/controllers/room_controller_test.exs

```diff
@@ -10,6 +10,8 @@ defmodule JellyfishWeb.RoomControllerTest do
     server_api_token = Application.fetch_env!(:jellyfish, :server_api_token)
     conn = put_req_header(conn, "authorization", "Bearer " <> server_api_token)
 
+    delete_all_rooms(conn)
```

Comment:
What about going even further and raise if some other test didn't do cleanup correctly? Does it make any sense to you? If not I would just move `delete_all_rooms(conn)` into `setup_all` so that we call it only once, before executing any test

```suggestion
setup_all %{conn: conn} do
  conn = get(conn, ~p"/room")
  response = json_response(conn, :ok)
  assert length(response["data"]) == 0
end
```

Reply:
It makes sense that all tests should clean their state after tests. But it won't be very pleasant to debug this problem in the future, and also, it is very easy to introduce it. Without this fix, 1 on 30 tests were failing, so we easily could overlook that our change introduces RC in tests. It is a little bit problem with this test `list all rooms` that this test could be affected by other tests.